profitbricks/src/main/java/org/jclouds/profitbricks/domain/DataCenter.java

```diff
@@ -38,11 +40,13 @@
 
    @Nullable public abstract Location location();
 
-//   @Nullable public abstract List<Server> servers();
-//   @Nullable public abstract List<Storage> storages();
+   @Nullable public abstract List<Server> servers();
+   
+   @Nullable public abstract List<Storage> storages();
 //   @Nullable public abstract List<LoadBalancer> loadBalancers();
-   public static DataCenter create(String id, String name, int version, ProvisioningState state, Location location) {
-      return new AutoValue_DataCenter(id, name, version, state, location);
+   public static DataCenter create(String id, String name, int version, ProvisioningState state, Location location, List<Server> servers,
+           List<Storage> storages) {
+      return new AutoValue_DataCenter(id, name, version, state, location, servers, storages);
```

Comment:
To enforce immutability it would be better to set the servers and storages with `ImmutableList.copyOf(servers)`.